flake.nix

```diff
@@ -31,6 +30,30 @@
             })
           ] ++ lib.optionals (stdenv.isLinux) [libudev-zero];
         };
+        stellarcli = stellardev // {
+          cargoLock = {
+            lockFile = ./Cargo.lock;
+          };
+
+          cargoLock.outputHashes = {
+              # The only git+https dependency in Cargo.lock
+              "stellar-rpc-client-21.4.0" = "sha256-ue7Ynux9YaDP3f/XkHz2OPd2g0iCX5R0yS5SaVHEYxQ";
```

Comment:
The used of a git dep is temporary until its released. Do we need to maintain this list whenever temporarily using git deps?